### PR TITLE
Fix Aqua related testing

### PR DESCRIPTION
**Pull request by Tokazama**
Right now all tests are failing because of the one line `Aqua.test_all(Static, piracy = false)`. I tried to look at what other SciML repos are doing to fix this but it would probably be best of @ChrisRackauckas or some other SciML expert ensures this PR goes about this correctly.

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -2,7 +2,14 @@ using Static, Aqua
 using Static: Zero
 using Test
 
-Aqua.test_all(Static, piracy = false)
+@testset "Aqua" begin
+    Aqua.test_ambiguities(Static, recursive = false)
```
Comment (ArnoStrouwen) [inline test/runtests.jl:6]:
```suggestion
    Aqua.find_persistent_tasks_deps(Static)
    Aqua.test_ambiguities(Static, recursive = false)
```

File: test/runtests.jl [hunk 2]
```diff
@@ -2,7 +2,14 @@ using Static, Aqua
...
+    Aqua.test_deps_compat(Static)
+    Aqua.test_project_extras(Static)
```
Comment (ArnoStrouwen) [inline test/runtests.jl:8]:
```suggestion
    Aqua.test_piracies(Static, broken = true)
    Aqua.test_project_extras(Static)
```

**Review by ArnoStrouwen — CHANGES_REQUESTED** [review]
The way to pass the keywords to `test_all` is pretty convoluted and error-prone in my opinion. These separate calls lead to fewer bugs.

**Comment by Tokazama** [comment]
Thanks @ArnoStrouwen!

**Pull request merged**